src/Network/Network/help/Start-AzVirtualNetworkGatewayConnectionPacketCapture.md

```diff
@@ -0,0 +1,213 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/start-azvirtualnetworkgatewayconnectionpacketcapture
+schema: 2.0.0
+---
+
+# Start-AzVirtualNetworkGatewayConnectionPacketCapture
+
+## SYNOPSIS
+Starts Packet Capture Operation on a Virtual Network Gateway Connection.
+
+## SYNTAX
+
+### ByName (Default)
+```
+Start-AzVirtualNetworkGatewayConnectionPacketCapture -ResourceGroupName <String> -Name <String>
+ [-FilterData <String>] [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByInputObject
+```
+Start-AzVirtualNetworkGatewayConnectionPacketCapture -InputObject <PSVirtualNetworkGatewayConnection>
+ [-FilterData <String>] [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Start-AzVirtualNetworkGatewayConnectionPacketCapture -ResourceId <String> [-FilterData <String>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Starts Packet Capture Operation on a Virtual Network Gateway Connection.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Start-AzVirtualNetworkGatewayConnectionPacketCapture -ResourceGroupName "PktCaptureTestSite2RG" -Name "PktCaptureTestSite2Site1Cn"
+
+Code              : Succeeded
+EndTime           : 10/1/2019 12:52:37 AM
+StartTime         : 10/1/2019 12:52:25 AM
+ResultsText       :
+ResourceGroupName : PktCaptureTestSite2RG
+Location          : centraluseuap
+ResourceGuid      : ac70028f-5b88-4ad4-93d3-0b9a9172c382
+Type              :
+Tag               :
+TagsTable         :
+Name              : PktCaptureTestSite2Site1Cn
+Etag              :
+Id                :
+```
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FilterData
+Filter options for start packet capture on virtual network gateway connection.
```

Comment:
Please provide at least one sample value here, or in the examples section, at least ne example of using FilterData

Reply:
Sure. I will add.

Reply:
Added FilterData scenario in tests.